src/core.js

```diff
@@ -138,8 +138,20 @@ RelativeFormat.prototype._compileMessage = function (units) {
     return new IntlMessageFormat(message, locales);
 };
 
-RelativeFormat.prototype._format = function (date) {
-    var now = dateNow();
+RelativeFormat.prototype._format = function (date, options) {
+    var now;
+    if (options && "now" in options) {
```

Comment:
The following needs to be valid, but in the current code will throw.

```js
relativeFormat.format(new Date(), {now: undefined});
```

Change to:

```js
var now = options && options.now !== undefined ? options.now : dateNow();
```